chore: first docs cleanup (#14)
humanitec-architecture/resource-packs-aws: README.md

---

> The AWS Resource Pack contains various reference implementation of connecting AWS with [Humanitec](https://humanitec.com/).

Reviewer: I suggest being more specific here and name the product, i.e. "the Humanitec Platform Orchestrator". "Humanitec" is the name of the company and no longer synonymous with the Orchestrator now that we're multi-product.

You can find more details about Resource Packs and how to use them [inside the Humanitec documentation](https://developer.humanitec.com/platform-orchestrator/resources/resource-packs/).

The following resources are included:

* [dns/basic](./humanitec-resource-defs/dns/basic): A `dns` resource using Route 53.
* [iam-policy/ecr-create-repository](./humanitec-resource-defs/iam-policy/ecr-create-repository): IAM policy for to create ECR repositories.
* [iam-policy/s3](./humanitec-resource-defs/iam-policy/s3): IAM policy for for an S3 bucket.
* [iam-policy/sqs](./humanitec-resource-defs/iam-policy/sqs): IAM policy for an SQS queue.
* [iam-role/service-account](./humanitec-resource-defs/iam-role/service-account): An assumable IAM role, by a k8s service account.
* [k8s/service-account](./humanitec-resource-defs/k8s/service-account): A `k8s-service-account` linked to an IAM role.

> * [rds/aurora](./humanitec-resource-defs/redis/basic): A `postgres` or `mysql` resource using AWS RDS Aurora.

Reviewer: ```suggestion
* [rds/aurora](./humanitec-resource-defs/rds/aurora): A `postgres` or `mysql` resource using AWS RDS Aurora.
```

> * [rds/basic](./humanitec-resource-defs/redis/basic): A `postgres` or `mysql` resource using AWS RDS.

Reviewer: ```suggestion
* [rds/basic](./humanitec-resource-defs/rds/basic): A `postgres` or `mysql` resource using AWS RDS.
```

* [redis/basic](./humanitec-resource-defs/redis/basic): A `redis` resource using AWS ElastiCache.
* [s3/basic](./humanitec-resource-defs/s3/basic): A basic `s3` bucket.
* [sqs/basic](./humanitec-resource-defs/sqs/basic): A basic `sqs` queue.
* [workload/service-account](./humanitec-resource-defs/workload/service-account): Inject a k8s service account into a `workload`.

> The `humanitec-resource-defs` directory includes the respective resource definitions.

Reviewer: ```suggestion
The `humanitec-resource-defs` directory includes the respective Resource Definitions.
```

The `modules` directory includes the Terraform modules used by the [Humanitec Terraform Driver](https://developer.humanitec.com/integration-and-extensions/drivers/generic-drivers/terraform/) to provision the actual resources.

> Checkout `examples/` for example application using the resource definitions listed above.

Reviewer: ```suggestion
Checkout `examples/` for example usage of the Resource Definitions listed above.
```
